src/plugin/WindowsKeyboardCodes.h

```diff
@@ -0,0 +1,301 @@
+#ifndef VK_UNKNOWN
+
+#define VK_UNKNOWN 0
+
+// Left mouse button
+#define VK_LBUTTON 0x01
+// Right mouse button
+#define VK_RBUTTON 0x02
+// Middle mouse button (three-button mouse)
+#define VK_MBUTTON 0x04
+#define VK_XBUTTON1 0x05
+#define VK_XBUTTON2 0x06
+
+#ifndef VK_BACK
```

Comment:
Why extra `#ifndef`s if there is already a global one?

Reply:
simply better safe then sorry. Preprocessor directives are never a bad thing, and don't impact performances. I just realized that the first 4 keycodes doesn't have the `#ifndef`, because I added them later.

Reply:
Maybe we can overcome this issue by directly inserting the keycodes into the switch-case of `custom_get_virtual_key()`. This saves a header file and as there are no other uses for these macros than the switch-case I don't see any problems of inserting these codes directly. The names returned from OBS look similar and thereby they are sufficient to express which keycode belongs to which key.

Reply:
i would prefer the mantain the OBS abstraction of keys in between, because of the fact that:
-The way OBS maps characters in future can change
-Microsoft has the bad habit to change often stuff